hack/update-bindata.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
```

Comment:
@thockin has done a bunch of work to get our Makefile doing stuff like this automatically. The Makefile's topological sort ensures that it is always done when necessary and (depending how fancy) not done when unnecessary.

I would strongly recommend that you produce either your own makefile for building the test job, or (probably better) add a rule in the main makefile.